.vib/kong/cypress/cypress/integration/kong_spec.js

```diff
@@ -68,10 +76,10 @@ it('allows adding a new service and route through admin API', () => {
       cy.wait(7000);
       cy.request({
         method: 'GET',
-        url: `${routes.newRoute.path}${random}`,
+        url: `${BASE_URL}/services/${services.newService.name}-${random}/routes`,
       }).then((response) => {
         expect(response.status).to.eq(200);
-        expect(response.body).to.contain(services.newService.upstreamContent);
+        expect(JSON.stringify(response.body)).to.contain(`${routes.newRoute.path}${random}`);
```

Comment:
We should remove `services.newService.upstreamContent` from the `services.json` file if we no longer use it.

Reply:
You are right, will do it!